backends-velox/src/main/resources/META-INF/services/org.apache.gluten.backendsapi.Backend

```diff
@@ -1 +1 @@
-org.apache.gluten.backendsapi.velox.VeloxBackend
+org.apache.gluten.backendsapi.VeloxBackend
```

Comment:
I see CH backend's impl is located at package `org.apache.gluten.backendsapi.clickhouse`. Should we do such movement only for Velox backend?

https://github.com/apache/incubator-gluten/blob/972004395369224e572690b0fe53d90c19cd3a94/backends-clickhouse/src/main/resources/META-INF/services/org.apache.gluten.backendsapi.Backend#L1C1-L1C51

Reply:
Also, what's the point of doing the movement?

Reply:
I only want to change the API names but not include this folder change. Let me recover it back.